models/index.js

```diff
@@ -5,18 +5,21 @@ var path = require("path");
 var Sequelize = require("sequelize");
 var basename = path.basename(module.filename);
 var env = process.env.NODE_ENV || "development";
-var config = require(__dirname + "/../config/config.json")[env];
-var db = {};
+require("dotenv").config();
 
-if (config.use_env_variable) {
-  var sequelize = new Sequelize(process.env[config.use_env_variable]);
+//var config = require(__dirname + "/../config/config.json")[env];
+var db = {};
```

Comment:
where does db get reinitialized? Why do we need it as an empty object here? Does it get reinitialized? If, so make it a let.